lore/technology/drone_manufacturer.mediawiki

```diff
@@ -0,0 +1,16 @@
+=History=
+Due to advancements in the field of macro-scale 3D printing after the field of nanotechnology was largely abandoned due to numeous scandals, manufacturing delicate, yet strong metal drone casings could be done reliably in an automated assembly process. Using this as a starting point, NanoTrasen decided to devote a substantial portion of their research and development facilities to achieving the goal of fully-automated maintenance and construction robots by 2520.
+
+The largest issue with designing the drones proved to be commonsense reasoning, as while construction was a relatively simple process, the sensorimotor component of the drones proved elusive. At the time, NanoTrasen drones were capable of complex reasoning, conversation, calculation, and planning, but failed to provide human-like performance on tasks like image recognition and movement.
```

Comment:
"... to be common sense reasoning..."

The second sentence doesn't seem related to the first, and seems like a weird order of events. Image recognition and movement are the parts of robotics that we're working most closely with *now*. Why is it the last thing that NanoTrasen managed to figure out?

I'd change the failure to be in responding naturally to physical stimulus, or some core behavior that requires sensorimotor components.

Reply:
Image recognition is what we're working with now, but we SUCK at it. Majorly so. Essentially: We think it works good, but it don't. https://medium.com/bethgelab/neural-networks-seem-to-follow-a-puzzlingly-simple-strategy-to-classify-images-f4229317261f